### PR TITLE
add minutes from sig release from 2022-08-31

### PR DESCRIPTION
**Pull request by fkr**
Signed-off-by: Felix Kronlage-Dammers <fkr@hazardous.org>

### CODE REVIEW

**Review by tibeer — APPROVED** [review]
LGTM!

File: sig-release/20220831.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+SIG RELEASE
+
+Beware: the contents of this pad will be publically available and the resulting minutes stored on github:
```
Comment (garloff) [inline sig-release/20220831.md:3]: Do we want to drop this from the final minutes?

**Review by garloff — APPROVED** [review]
/lgtm

File: sig-release/20220831.md (hunk 1 above)
Comment (fkr) [reply sig-release/20220831.md]: thanks @garloff!

**Pull request merged**